draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: >
-  Internet X.509 Public Key Infrastructure: Algorithm Identifiers for ML-DSA
+  Internet X.509 Public Key Infrastructure - Algorithm Identifiers
+  for the Module-Lattice-Based Digital Signature Algorithm (ML-DSA)
```

Comment:
Note: Annoyingly NIST uses `Standard` instead of `Algorithm` in the title of FIPS 204. I think using *Algorithm* here makes more sense.

Reply:
I agree, I wanted to use `Standard` also but felt it was more correct to abide by the `DSA` part!